platformio.ini

```diff
@@ -637,12 +639,14 @@ lib_ignore  = LiquidCrystal, LiquidTWI2, Adafruit NeoPixel, TMCStepper, U8glib-H
 #
 [env:STM32F070RB_malyan]
 platform    = ststm32
+platform_packages = framework-arduinoststm32@${common.arduinoststm32_ver}
 board       = malyanM200v2
 build_flags = -DSTM32F0xx -DUSBCON -DUSBD_VID=0x0483 '-DUSB_MANUFACTURER="Unknown"' '-DUSB_PRODUCT="ARMED_V1"' -DUSBD_USE_CDC -DHAL_PCD_MODULE_ENABLED
   -O2 -ffreestanding -fsigned-char -fno-move-loop-invariants -fno-strict-aliasing -std=gnu11 -std=gnu++11
+  -DCUSTOM_STARTUP_FILE
```

Comment:
I honestly don't know if adding `-DCUSTOM_STARTUP_FILE` was the right solution here, but it was the only way I could get it to build. This board's variant has it's own startup file which conflicts with the default startup file in the framework.